packages/vaadin-grid/test/resizing.test.js

```diff
@@ -120,6 +120,22 @@ describe('resizing', () => {
     expect(getPhysicalItems(grid).length).to.be.above(0);
   });
 
+  it('should have body rows after items reset and repopulated', async () => {
+    grid = fixtureSync(`
+      <vaadin-grid>
+        <vaadin-grid-column path="value"></vaadin-grid-column>
+      </vaadin-grid>`);
```

Comment:
Consider moving this test and the one above to a separate test suite (without `test-component`). Not a blocker.

Reply:
There was a `height by rows` suite in the file but for some reason, the new tests weren't inside of it. Done.